pkg/registries/registries.go

```diff
@@ -385,7 +372,8 @@ func IsValidRegistriesConfScope(scope string) bool {
 // RejectMultiUpdateMirrorSetObjs returns error if icsp objects exist with imagedigestmirrorset or imagetagmirrorset objects
 // to avoid existing mirror settings get updated by others
 func RejectMultiUpdateMirrorSetObjs(icspRules []*apioperatorsv1alpha1.ImageContentSourcePolicy,
```

Comment:
BTW should this function be removed, so that callers, if any, notice they should change their implementation?

Reply:
Good idea. MCO is the only caller, so that is safe.